.github/workflows/continuous_delivery.yaml

```diff
@@ -18,13 +18,16 @@ jobs:
     name: Test
     runs-on: ubuntu-latest
     steps:
-    - name: Setup
+    - name: Checkout
+      uses: actions/checkout@v2
+
+    - name: Setup go
       uses: actions/setup-go@v2
       with:
         go-version: '1.16'
 
-    - name: Checkout
-      uses: actions/checkout@v2
+    - name: Set up docker buildx
+      uses: docker/setup-buildx-action@v1
```

Comment:
first off, you are doing some sophisticated stuff in here between the workflow and Makefile. that said since we are pushing images here i don't see we need to make any changes to the workflow.

however if we do want to start pushing here, i think i found a guide:

https://github.com/docker/build-push-action/blob/master/docs/advanced/multi-platform.md